pkgs/development/tools/iaca/2.1.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, makeWrapper, requireFile, patchelf, gcc, unzip }:
+assert stdenv.system == "x86_64-linux";
+with stdenv.lib;
+
+stdenv.mkDerivation {
+  name = "iaca-2.1";
```

Comment:
Why this particular version?

Reply:
2.1 is the last version to support NHM/WSM arches; 3.0 has everything else.

Reply:
Can you add this add as a comment. Otherwise people might remove this version at some point.

Reply:
sure, how's this? (50dee8d)